exporter/collector/integrationtest/testcases/testcases_metrics.go

```diff
@@ -135,12 +135,9 @@ var MetricsTestCases = []TestCase{
 			cfg.MetricConfig.GetMetricName = googlemanagedprometheus.GetMetricName
 			cfg.MetricConfig.MapMonitoredResource = googlemanagedprometheus.MapToPrometheusTarget
 			cfg.MetricConfig.ExtraMetrics = func(m pmetric.Metrics) pmetric.ResourceMetricsSlice {
-				extraMetrics := googlemanagedprometheus.AddTargetInfoMetric(m)
-				scopeInfoMetrics := googlemanagedprometheus.AddScopeInfoMetric(m)
-				for i := 0; i < scopeInfoMetrics.Len(); i++ {
-					scopeInfoMetrics.At(i).ScopeMetrics().MoveAndAppendTo(extraMetrics.At(i).ScopeMetrics())
-				}
-				return extraMetrics
+				googlemanagedprometheus.AddScopeInfoMetric(m)
+				googlemanagedprometheus.AddTargetInfoMetric(m)
```

Comment:
this ordering is important (first scope_info, then target_info), otherwise the `otel_scope_info` metric is also added to the new (empty) ScopeMetrics that holds `target_info` for a resource. I actually wasn't sure on this, but took an educated guess

Reply:
This shouldnt' matter if we adopt my recommendation above.